src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -1,45 +1,34 @@
 // @flow
 import * as React from 'react';
+
 import classNames from 'classnames';
 import noop from 'lodash/noop';
-import TetherComponent from 'react-tether';
-import { FormattedMessage } from 'react-intl';
+
+import { BaseCommentMenuWrapper } from './components/BaseCommentMenuWrapper';
+import { BaseCommentMetadata } from './components/BaseCommentMetadata';
+import { COMMENT_STATUS_RESOLVED } from '../../../../constants';
+
 import ActivityError from '../common/activity-error';
 import ActivityMessage from '../common/activity-message';
-import ActivityStatus from '../common/activity-status';
-import ActivityTimestamp from '../common/activity-timestamp';
-import Avatar from '../Avatar';
-import Checkmark16 from '../../../../icon/line/Checkmark16';
 import CommentForm from '../comment-form';
 import CreateReply from './CreateReply';
-import DeleteConfirmation from '../common/delete-confirmation';
-import IconAnnotation from '../../../../icons/two-toned/IconAnnotation';
 import LoadingIndicator from '../../../../components/loading-indicator';
-import Media from '../../../../components/media';
-import messages from './messages';
-import Pencil16 from '../../../../icon/line/Pencil16';
 import RepliesToggle from './RepliesToggle';
-import Trash16 from '../../../../icon/line/Trash16';
-import UserLink from '../common/user-link';
-import X16 from '../../../../icon/fill/X16';
-import { ACTIVITY_TARGETS } from '../../../common/interactionTargets';
-import { COMMENT_STATUS_OPEN, COMMENT_STATUS_RESOLVED, PLACEHOLDER_USER } from '../../../../constants';
-import { MenuItem } from '../../../../components/menu';
-import { type OnAnnotationEdit, type OnCommentEdit } from './types';
+
 import type {
     ActionItemError,
     BoxCommentPermission,
     FeedItemStatus,
     Comment as CommentType,
 } from '../../../../common/types/feed';
-
 import type { GetAvatarUrlCallback, GetProfileUrlCallback } from '../../../common/flowTypes';
+import type { OnAnnotationEdit, OnCommentEdit } from './types';
 import type { SelectorItems, User } from '../../../../common/types/core';
 import type { Translations } from '../../flowTypes';
 
 import './BaseComment.scss';
-import './Replies.scss';
 import './Comment.scss';
+import './Replies.scss';
```

Comment:
Import order for CSS can have side effects. Is this needed?

Reply:
Oh, that's a good point. You're right that I was just naively alphabetizing. There was no effect, but not needed. I'll revert this.